udata_front/theme/gouvfr/assets/js/pages/ReusePublishingForm/Step1DescribeReuse.stories.ts

```diff
@@ -0,0 +1,128 @@
+import { withActions } from '@storybook/addon-actions/decorator';
+import { fn } from '@storybook/test';
+import type { Meta, StoryObj } from '@storybook/vue3';
+import Step1DescribeReuse from './Step1DescribeReuse.vue';
+
+const meta = {
+  title: 'Pages/ReusePublishingForm/Step1',
+  component: Step1DescribeReuse,
+  decorators: [withActions],
+  args: {
+    onNext: fn(),
+  }
+} satisfies Meta<typeof Step1DescribeReuse>;
+
+export default meta;
+
+const myUser = {
+  id: "someId",
+  first_name: "John",
+  last_name: "Doe",
+  avatar: "https://demo-static.data.gouv.fr/avatars/84/3194d831264f769fa817e58813d413-100.png",
+  organizations: [
+    {
+      acronym: null,
+      badges: [],
+      class: "Organization",
+      id: "65e9b7cf830c3b5a515ee4ed",
+      logo: "https://demo-static.data.gouv.fr/avatars/84/3194d831264f769fa817e58813d413-100.png",
+      logo_thumbnail: "https://demo-static.data.gouv.fr/avatars/84/3194d831264f769fa817e58813d413-100.png",
+      name: "My Organization",
+      page: "",
+      slug: "my-organization",
+      uri: "/"
+    },
+    {
+      acronym: null,
+      badges: [],
+      class: "Organization",
+      id: "65e9b7cf830c3b5a515ee4ed",
```

Comment:
Can you use a different id ? Otherwise the producer selector is showing both as selected